### PR TITLE
Add South Korea

### PR DESCRIPTION
**Pull request by systemcatch**
This needs lots of cleanup and fixes.

- [x] CO2 value for unknown.
- [x] Config setup (zones, readme, en lang).
- [x] Cleanup all notes.
- [x] Make it more DRY.
- [x] Hydro capacity check.
- [x] Docstrings.
- [x] Add hydro and battery to storage.
- [x] Test on mockserver.

ref #1563

### CODE REVIEW

File: parsers/KR.py [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+    production = {
+                  'production': {
+                    'nuclear': nuclear,
+                    'hydro': hydro,
+                    'unknown': unknown
+                  },
+                  'source': 'khnp.co.kr, kpx.or.kr',
+                  'zoneKey': zone_key,
+                  'datetime': dt_aware.datetime,
+                  'storage': {}
```
Comment (corradio) [inline parsers/KR.py]: better just remove that line if you don't plan on putting storage

**Comment by corradio** [comment]
Seems pretty cool if indeed wind/solar is such a small part

File: parsers/KR.py (hunk 1 above)
Comment (systemcatch) [reply parsers/KR.py]: actually according to Alex there's pumped storage and fuel cells (battery?) so I'll add them

**Comment by systemcatch** [comment]
> Seems pretty cool if indeed wind/solar is such a small part

I think it's a good start, we can build on this further if/when more data becomes available. Also SK has no exchanges atm so that will stop it effecting the rest of the dataset.

**Comment by systemcatch** [comment]
Seems to be basically working, looks a little funny with the coal/gas capacities included.

SK

File: config/co2eq_parameters.json [hunk 2]
```diff
@@ -406,6 +406,14 @@
           "value": 413.76308999360253
         }
       },
+      "KR": {
+        "unknown": {
+          "_comment": "Source: Derived from 2016 IEA data",
```
Comment (brunolajoie) [inline config/co2eq_parameters.json]:
```suggestion
          "_comment": "Source: Derived from 2016 annual average by IEA",
```

**Comment by systemcatch** [comment]
@corradio good to go!

**Pull request merged**